custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -18,10 +18,12 @@
 
 
 def validate_phone_number(string_value):
-    if string_value is None or string_value.strip() == '':
+    if string_value is None or string_value.strip() in ['', '0']:
         return ''
     else:
-        return int(string_value)
+        phone_number = str(int(string_value))
+        assert len(phone_number) == 10
```

Comment:
just wanted to reiterate that in order to work with messaging, numbers have to be in international format. if all the numbers are exactly 10 digits, then all we should have to do is add '91' to the front of them to put them in international format for India.

how else is this phone number used? if you need to keep a version of it in national format (without the '91'), you'd have to put it in another case property since `contact_phone_number` should always be in international format